app/src/routes/routesMap.js

```diff
@@ -366,7 +366,9 @@ export const onBeforeRouteChange = (dispatch, getState, { action }) => {
       !hashProjectSlug &&
       (isAssignedToTargetOrganization || assignmentNotRequired)
     ) {
-      dispatch(fetchOrganizationBySlugAction(hashOrganizationSlug));
+      if (nextPageType !== ORGANIZATION_PROJECTS_PAGE) {
+        dispatch(fetchOrganizationBySlugAction(hashOrganizationSlug));
+      }
```

Comment:
For now the organization request is sent from `ORGANIZATION_PROJECTS_PAGE` via `prepareActiveOrganizationProjectsAction`.
What if the request for organization will be also send from another page in the future? I suppose it's possible when new page on organization level will be added.
So I suggest to leave `fetchOrganizationBySlugAction(hashOrganizationSlug)` as required action here and update `prepareActiveOrganizationProjects` from `src/controllers/organizations/organization/sagas.js` to fix the bug.